modules/ROOT/pages/platform/metrics-integration.adoc

```diff
@@ -0,0 +1,430 @@
+* Tenant is of type `enterprise`,
+* specified instance belongs to the specified tenant.
+
+[#revoke-access-to-metrics]
+=== Revoke access to metrics
+
+To revoke user's access to metrics of a specific tenant, remove the user from that tenant in https://console.neo4j.io/#user-management["User Management"^]. After that, the user will still exist but its connection to the tenant will be removed.
+
+____
+*_NOTE_:* The revocation described above takes effect after the authorization caches expire, which takes approximately 5 minutes.
+It results in HTTP 401 being returned, along with the message `User doesn't have access to Metrics resources.`
+____
+
+____
+*_NOTE_:* If, instead of removing the user, only the Aura API credentials used to retrieve metrics are removed, the revocation will take effect only after the tokens issued with these credentials expire, as no new token can be issued anymore. Currently used token expiration time is 1 hour.
+____
+
+[#metric-labels]
+=== Metric labels
+
+Depending on the metric, following labels are applied:
+
+* `aggregation`: the aggregation used to calculate the metric value, set on every metric. Since the Neo4j instance is deployed as a Neo4j cluster, aggregations are performed to combine values from all relevant cluster nodes. Currently used aggregations: `MIN`, `MAX`, `AVG` and `SUM`.
+* `instance_id`: the Aura instance ID the metric is reported for, set on every metric.
+* `database`: the name of the Neo4j database the metric is reported for. Currently always set to `neo4j`.
+
+Example:
+
+[,shell]
+----
+# HELP neo4j_database_count_node The total number of nodes in the database.
+# TYPE neo4j_database_count_node gauge
+neo4j_database_count_node{aggregation="MAX",database="neo4j",instance_id="78e7c3e0"} 778114.000000 1711462853000
+----
+
+[#looking-up-metric-name-in-neo4j-aura-advanced-metrics]
+=== Looking up metric name in Neo4j Aura Advanced Metrics
+
+In Neo4j Aura Advanced Metrics, it is possible to find out the metric name that corresponds to the chart, by using the chart menu item "Metrics Integration" as shown below.
+
+image::advanced_metrics.png[]
+
+[#metric-scrape-interval]
+== Metric scrape interval
+
+Recommended scrape interval for metrics is in the range of 30 seconds up to 2 minutes, depending on requirements. The metrics endpoint caches metrics for 30 seconds.
+
+[#example-of-using-prometheus]
+== Example of using Prometheus
+
+[#install-prometheus]
+=== Install Prometheus
+
+E.g. by getting a tarball from https://prometheus.io/docs/prometheus/latest/installation/[^]
+
+[#configure-prometheus]
+=== Configure Prometheus
+
+To monitor one or more instances, add a section to the Prometheus configuration file `prometheus.yml`.
+
+The simplest way is to copy the configuration section proposed in https://console.neo4j.io/#metrics-integration["Metrics Integration"^], by using the button right to the endpoint URL, as shown below.
```

Comment:
```suggestion
Copy the configuration section proposed in link:https://console.neo4j.io/#metrics-integration[Metrics Integration^], as shown.
```
We want to avoid describing how to click buttons in the docs.